include/envoy/server/configuration.h

```diff
@@ -147,6 +147,11 @@ class Admin {
   virtual const std::string& accessLogPath() PURE;
 
   /**
+   * @return const std::string& the admin profiler path.
```

Comment:
Actually, it might be more semantically correct to call this profilePath (and similar renames elsewhere). It's where the profile is emitted, not the profiler itself. The comment might then be "@return const std::string& profiler output path."